exercises/space-age/src/example.h

```diff
@@ -1,18 +1,18 @@
 #ifndef SPACE_AGE_H
 #define SPACE_AGE_H
 
-/* Number of seconds in a Earth year */
-#define EARTH 31557600
+typedef enum planets {
+   mercury = 0,
+   venus,
+   earth,
+   mars,
+   jupiter,
+   saturn,
+   uranus,
+   neptune,
+   max_planet = neptune
```

Comment:
This is not necessary (and makes it more likely to break if they ever make Pluto a planet again) because this is the automatic behavior.